spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -128,28 +128,42 @@ def mock_column(name, type, options = {})
     end
   end
 
-  describe '#parse_options' do
+  describe '.parse_options' do
     let(:options) do
       {
         root_dir: '/root',
         model_dir: 'app/models,app/one,  app/two   ,,app/three'
       }
     end
 
-    it 'sets @root_dir' do
+    before :each do
       AnnotateModels.send(:parse_options, options)
-      expect(AnnotateModels.instance_variable_get(:@root_dir)).to eq('/root')
     end
 
-    it 'sets @model_dir separated with a comma' do
-      AnnotateModels.send(:parse_options, options)
-      expected = [
-        'app/models',
-        'app/one',
-        'app/two',
-        'app/three'
-      ]
-      expect(AnnotateModels.instance_variable_get(:@model_dir)).to eq(expected)
+    describe '@root_dir' do
+      subject do
+        AnnotateModels.instance_variable_get(:@root_dir)
+      end
+
+      it 'sets @root_dir' do
+        expect(AnnotateModels.instance_variable_get(:@root_dir)).to eq('/root')
```

Comment:
Can `expect(subject)...` also be used here?

Reply:
If so, could it just be `is_expected` instead of `expect(subject)`?

Reply:
I fixed it using `is_expected`.